Refactor API exposure tutorials (#15332)
kyma-project/kyma: docs/03-tutorials/00-api-exposure/apix-03-expose-workload-apigateway.md

---

This tutorial shows how to expose service endpoints and configure different allowed HTTP methods for them using API Gateway Controller.
   > **CAUTION:** Exposing a workload to the outside world is always a potential security vulnerability, so tread carefully. In a production environment, always secure the workload you expose with [OAuth2](./apix-05-expose-and-secure-workload-oauth2.md) or [JWT](./apix-08-expose-and-secure-workload-jwt.md).

The tutorial may be a follow-up to the [Set up a custom domain for a workload](./apix-02-setup-custom-domain-for-workload.md) tutorial.

## Prerequisites

> This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create one of those, follow the [Create a workload](./apix-01-create-workload.md) tutorial.

Reviewer: same as before not sure about the Function part.

Reviewer: my bad, didn't see the Function section.

1. Export the following value as an environment variable:

   ```bash

>    export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME} #This is a Kyma domain or your custom subdomain, for example, api.mydomain.com

Reviewer 2: can we move this comment to a note below?

   export GATEWAY=$NAMESPACE/httpbin-gateway
   ```

>    >**NOTE:** If you don't want to use your custom domain but a Kyma domain, use the following Kyma Gateway: `kyma-system/kyma-gateway`

Reviewer 2: ```suggestion
   >**NOTE:** If you want to use Kyma domain, export it as `DOMAIN_TO_EXPOSE_WORKLOADS` and use the following Kyma Gateway: `kyma-system/kyma-gateway`
```
or something similar. this way we will get rid of suggestions in comments. Please change that consistently everywhere
